pycbc/dq.py

```diff
@@ -29,7 +29,7 @@
 import numpy
 from astropy.utils.data import download_file
 from ligo.segments import segmentlist, segment
-
+from pycbc.frame.losc import _get_run
```

Comment:
If this is now something we are calling from outside of this module then `_get_run` -> `get_run`